app/controllers/api/base_controller.rb

```diff
@@ -1,13 +1,7 @@
 module Api
-  class BaseController < ApplicationController
+  class BaseController < ActionController::Base
     protect_from_forgery with: :null_session
 
```

Comment:
Looks like you’ll need to include `set_current_tenant_through_filter` at the top of this controller